### PR TITLE
boveri Follow_up node updates

### PR DESCRIPTION
**Pull request by ganjigajanan**

### CODE REVIEW

**Review by wwysoc2 — APPROVED** [review]
lgtm

File: gdcdictionary/schemas/follow_up.yaml [hunk 1]
```diff
@@ -1278,6 +1286,8 @@ properties:
     $ref:
       - "_terms.yaml#/dlco_ref_predictive_percent/common"
     type: number
+    maximum: 100
```
Comment (wteouchicago) [inline gdcdictionary/schemas/follow_up.yaml:1289]: just noting this isn't in the DICT-98 ticket, but makes sense to include.

File: gdcdictionary/schemas/follow_up.yaml [hunk 2]
```diff
@@ -1611,11 +1669,13 @@ properties:
     $ref:
       - "_terms.yaml#/nadir_cd4_count/common"
     type: number
+    minimum: 0
 
   pancreatitis_onset_year:
     $ref:
       - "_terms.yaml#/pancreatitis_onset_year/common"
     type: integer
+    minimum: 1900
```
Comment (wteouchicago) [inline gdcdictionary/schemas/follow_up.yaml:1678]: noting this isn't in ticket, but good to include

File: gdcdictionary/schemas/follow_up.yaml [hunk 3]
```diff
@@ -2287,6 +2352,88 @@ properties:
...
+          - "_terms.yaml#/no/common"
+      Not Reported:
+        $ref:
+          - "_terms.yaml#/not_reported/common"
+
+  undescended_testis_corrected_age:
+    $ref:
+      - "_terms.yaml#/undescended_testis_corrected_age/common"
+    type: integer
+    minimum: 0
```
Comment (wteouchicago) [inline gdcdictionary/schemas/follow_up.yaml:2377]: I guess in theory we could have a max of 89 here...

**Review by wteouchicago — APPROVED** [review]

**Pull request merged**